engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -822,3 +842,12 @@ definitions:
       jobs:
         type: integer
         description: The number of currently running jobs
+  VersionResponse:
+    description: 'Provides the version of the engine running inside the system'
+    properties:
+      cromwell:
+        type: string
+        description: The version of the Cromwell Engine
+      wdl4s:
+        type: string
+        description: The version of the Wdl4S beign used by Cromwell
```

Comment:
Do we really want the wdl4s version ? wdl4s is just a dependency of cromwell as many others IMO.
The wdl version (when well defined) would be better.